### DISCUSSION TITLE
How can i remove or not apply tailwindcss style to a specific elemement

### DISCUSSION THREAD

**Discussion by abkarim** [tailwindlabs/tailwindcss#10787] [Help]
I using tailwindcss in my project.
Tailwindcss removed all default style for my project.

But i have an spefiic div that doesn't require these to reset browser default styles. I need broser default style for that div 

Can i do any configuration to not apply tailwindcss by applying a ignore class. 
Or something like that ?

**Comment by wongjn** [comment]
If you are running Tailwind as a PostCSS plugin, you could add a PostCSS plugin after Tailwind that adds an extra portion to all Tailwind rules' selectors that scopes them to not apply to the `<div>`.

**Comment by andrepichardo** [comment]
Hi @abkarim! 

Tailwind CSS doesn't have a built-in feature to "opt-out" of the preflight styles for just an element, unfortunately.

An option would be to use the `all` CSS property ([MDN page](https://developer.mozilla.org/en-US/docs/Web/CSS/all)). I think it can solve some of your needs, especially `all: initial`. You could try giving a class to that specific `div` and apply the mentioned CSS property and check if that works.

Hope this answer helps! 👍

**Reply by jianglei-zll** [reply]
Thank you for helping me
